cocos/audio/include/AudioEngine.h

```diff
@@ -284,11 +289,12 @@ class EXPORT_DLL AudioEngine
     /**
     * Preload audio file.
     * @param filePath The file path of an audio.
+    * @param callback A callback which will be called after loading is finished.
     */
-    static void preload(const std::string& filePath);
+    static void preload(const std::string& filePath, AudioPreloadCallback callback = nullptr);
```

Comment:
please use overload instead of default arguments, add another `static void preload(const std::string& filePath, AudioPreloadCallback callback );` is better.
For more discussion, please refer to https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md#default-arguments